Marlin/language_da.h

```diff
@@ -182,8 +183,8 @@
 #define MSG_THERMAL_RUNAWAY                 "Temp løber løbsk"
 #define MSG_ERR_MAXTEMP                     "Fejl: Maks temp"
 #define MSG_ERR_MINTEMP                     "Fejl: Min temp"
-#define MSG_ERR_MAXTEMP_BED                 "Fejl: Maks P temp"
-#define MSG_ERR_MINTEMP_BED                 "Fejl: Min P temp"
+#define MSG_ERR_MAXTEMP_BED                 "Fejl: Maks Plsde temp"
+#define MSG_ERR_MINTEMP_BED                 "Fejl: Min Plade temp"
```

Comment:
Remember, anything over 18 or 20 characters will be cut off. Try to keep the important part within the first 18 characters.